Components/Alert/alert.html

```diff
@@ -0,0 +1,293 @@
+                        <a
+                            href="/Components/Button/button.html"
+                            class="drawer-link"
+                            >Button</a
+                        >
+                        <a href="/Components/Card/card.html" class="drawer-link"
+                            >Card</a
+                        >
+                        <a
+                            href="/Components/Image/image.html"
+                            class="drawer-link"
+                            >Image</a
+                        >
+                        <a
+                            href="/Components/Input/input.html"
+                            class="drawer-link"
+                            >Input</a
+                        >
+                    </div>
+                </aside>
+            </div>
+
+            <div>
+                <div class="component-content-container">
+                    <div class="component-nav-container">
+                        <a
+                            href="/Components/Avatar/avatar.html"
+                            class="component-nav-link"
+                        >
+                            <i class="material-icons component-nav-arrow">
+                                arrow_left
+                            </i>
+                            <span>Avatar</span>
+                        </a>
+                        <a
+                            href="/Components/Badge/badge.html"
+                            class="component-nav-link"
+                        >
+                            <span>Badge</span
+                            ><i class="material-icons component-nav-arrow">
+                                arrow_right
+                            </i>
+                        </a>
+                    </div>
+
+                    <h1 class="component-heading">Alert</h1>
+
+                    <h3 class="component-subheading">
+                        Alert component CSS - import url
+                    </h3>
+                    <iframe
+                        src="https://www.thiscodeworks.com/embed/61fc003b6ab14d0015c32d8e"
+                        style="
+                            width: 54rem;
+                            height: 6.875rem;
+                            margin: 1em 0;
+                            max-width: 100%;
+                        "
+                        frameborder="0"
+                    ></iframe>
+
+                    <h3 class="component-subheading">
+                        Alert component CSS - HTML link tag
+                    </h3>
+                    <iframe
+                        src="https://www.thiscodeworks.com/embed/61fc00966ab14d0015c32d8f"
+                        style="
+                            width: 54rem;
+                            height: 6.875rem;
+                            margin: 1em 0;
+                            max-width: 100%;
+                        "
```

Comment:
avoid inline style

Reply:
Actually for the iframes the class styling was not working i spent so many hours trying even asked on group, finally had to do this only for iframes but.